maven-model-builder/src/main/java/org/apache/maven/feature/Features.java

```diff
@@ -0,0 +1,63 @@
+package org.apache.maven.feature;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * Centralized class for feature information
+ * 
+ * @author Robert Scholte
+ * @since 3.7.0
+ */
+public final class Features
+{
+    private Features() 
+    {
+    }
+    
+    private static final Feature BUILDCONSUMER = new Feature( "maven.experimental.buildconsumer", "true" );
+    
+    public static Feature buildConsumer()
+    {
+        return BUILDCONSUMER;
+    }
+    
+    /**
+     * Represents some feature
+     * 
+     * @author Robert Scholte
+     * @since 3.7.0
+     */
+    public static class Feature
+    {
+        private final boolean active;
+
+        Feature( String name, String defaultValue )
+        {
+            active = "true".equals( System.getProperty( name, defaultValue ) );
```

Comment:
Why not use `Boolean#getBoolean(java.lang.String)`?

Reply:
`Boolean#getBoolean` doesn't support a default value